File: competitive_programming/codeforces/06_javascript/50a_domino_piling.js
function dominoPiling(m, n) {
  return Math.floor((m * n) / 2);
}

if (require.main === module) {
  if (dominoPiling(2, 4) !== 4 || dominoPiling(3, 3) !== 4) throw new Error("domino piling failed");
  console.log("50A domino piling ok");
}

File: competitive_programming/codeforces/09_rust/50a_domino_piling.rs
fn domino_piling(m: i64, n: i64) -> i64 {
    m * n / 2
}

fn main() {
    assert_eq!(domino_piling(2, 4), 4);
    assert_eq!(domino_piling(3, 3), 4);
    println!("50A domino piling ok");
}
